pkg/cloud/services/compute/bastion.go

```diff
@@ -23,12 +23,18 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/pkg/record"
 )
 
-func (s *Service) DeleteBastion(openStackCluster *infrav1.OpenStackCluster, serverID string) error {
-	instance, err := s.GetInstance(serverID)
+func getBastionName(clusterName string) string {
```

Comment:
Can we ensure we don't need two places where we calculate the bastionname? I.e. just call Delete/CreateBasiton and the compute package handles it from there?

Reply:
I will update the name back, but existing code has the issue. The another PR might be solve the issue.

Reply:
Okay fine for me